android/src/main/java/com/magicleap/magicscript/scene/nodes/button/UiButtonNode.kt

```diff
@@ -116,46 +128,40 @@ open class UiButtonNode(
             setTextPadding(textPaddingHorizontal, textPaddingVertical)
 
             setCharactersSpacing(charactersSpacing)
+            onPressedChangeListener = { pressed ->
+                val z =
+                    if (pressed) {
+                        ANIMATED_Z
+                    } else {
+                        0f
+                    }
+                localPosition = Vector3(localPosition.x, localPosition.y, z)
```

Comment:
This will also not work, because you assign the `initialLocalPosition` only once at view setup, simple use the previous solution, but do `contentNonde.localPosition=...` instead `localPosition=...`